app/commands/put_content_with_links.rb

```diff
@@ -61,5 +63,15 @@ def delete_existing_links
     def protected_link_types
       ["alpha_taxons"]
     end
+
+    def protected_apps
+      # specialist-publisher is being converted to a "standard" Rails app
+      # that uses the publishing-api V2 endpoints. We don't know when the
+      # conversion completed and we want all the links of specialist-publisher
```

Comment:
Do you mean "when the conversion will be completed"?  Or has it completed already?